src/airflow/requirements.txt

```diff
@@ -1 +1,2 @@
 apache-airflow-providers-google
+gcsfs
```

Comment:
Does this module provide a wrapper for accessing Google Cloud Storage as a POSIX filesystem?

Reply:
Thank you, this is actually not required anymore

Reply:
Now removed